webui/react/src/components/WorkspaceCreateModal.tsx

```diff
@@ -258,8 +235,8 @@ const WorkspaceCreateModalComponent: React.FC<Props> = ({ onClose, workspaceId }
         }
 
         if (workspaceId) {
-          const response = await patchWorkspace({ id: workspaceId, ...body });
-          setWorkspace(response);
+          await patchWorkspace({ id: workspaceId, ...body });
+          workspaceStore.fetch();
```

Comment:
**chore:** I think this might need the `force` flag, otherwise it'll just skip the actual API call.
```
workspaceStore.fetch(undefined, true);
```

Reply:
✔️